### DISCUSSION TITLE
Bullhorn not returning the right results

### DISCUSSION THREAD

**Discussion by klemen-traven** [bullhorn/career-portal#442] [General]

**Comment by charlesabarnes** [comment]
Hello, please contact our support team.  They should be able to help with this.   This issue needs more details that is not appropriate to troubleshoot on this site

**Reply by klemen-traven** [reply]
The support team actually brought me here... so it looks like a dead end to me.

**Reply by charlesabarnes** [reply]
Heya, apologies about that, that is not the correct route.  I will reach out internally, but please reach back out as well and you can refer the analyst to this post

**Reply by charlesabarnes** [reply]
![image](https://user-images.githubusercontent.com/4327754/142241712-0c8968bc-4e69-4d49-9d14-88981815ce4e.png)


Just for reference, these are the guidelines for where an issue should generally go.  There are exceptions, but since you have the API calls, this is definitely in the "API" category
